shared/chat/conversations-list/container.js

```diff
@@ -69,7 +69,7 @@ class ConversationListContainer extends Component {
       const participants = participantFilter(conversation.get('participants'), this.props.you)
       const isSelected = this.props.selectedConversation === conversationIDKey
       const isMuted = conversation.get('muted')
-      const rekeyInfo = this.props.selectedConversation && this.props.rekeyInfos.get(conversationIDKey)
+      const rekeyInfo = this.props.rekeyInfos.get(conversationIDKey)
```

Comment:
Without this change you get a weird state where you only see rekey info when you have selected a conversation. That only happens the moment you click a conversation while the conversation list fades into the background. Looks really buggy.

Not sure why you'd want this in desktop. It would hide the fact you need to rekey until you open any conversation.